src/poliastro/twobody/orbit.py

```diff
@@ -368,12 +369,23 @@ def from_equinoctial(
 
     @classmethod
     def from_body_ephem(cls, body, epoch=None):
-        """Return osculating `Orbit` of a body at a given time.
 
-        """
+        """Return osculating `Orbit` of a body at a given time."""
+
         # TODO: https://github.com/poliastro/poliastro/issues/445
+
+        if (
+            body.name == "Pluto"
```

Comment:
Thanks for the PR @Himanshu-Garg. As said in #401, for the case of the Moon same happens and therefore some message should be raised. The problem is that 'Moon' is included in both `astropy` ephemerides: `builtin` and `de432s`. So therefore, here is my code suggestion:

```suggestion
            body.name == "Moon"
            or body.name.lower() not in solar_system_ephemeris.bodies
```